File: Client/Game_GoalFootball/src/Dto/GameCommand/GameBetDto.ts
namespace Dto{
    /**
     * 游戏投注Dto
     */
    export class GameBetDto{
        /**
         * 投注基数（每注金额）
         */
        Amount: number = 0;
        /**
         * 道具使用情况
         */
        Props: Array<any> = [];
    }
}

File: QIC.EGame.GameMachineService/QIC.EGame.GameMachineService.GoalFootball/Dtos/GameBetDto.cs
using System.Collections.Generic;
using QIC.EGame.GameMachineService.GoalFootball.Enums;

namespace QIC.EGame.GameMachineService.GoalFootball.Dtos
{
    /// <summary>
    /// 游戏下注的Dto
    /// </summary>
    public class GameBetDto
    {
        /// <summary>
        /// 投注基数（即每注金额）
        /// </summary>
        public decimal Amount { get; set; }
        /// <summary>
        /// 道具使用情况
        /// </summary>
        public List<int> Props { get; set; }
    }
}
